mlir/include/mlir/Dialect/OpenMP/OpenMPClauses.td

```diff
@@ -422,11 +455,11 @@ class OpenMP_IfClauseSkip<
   > : OpenMP_Clause</*isRequired=*/false, traits, arguments, assemblyFormat,
                     description, extraClassDeclaration> {
   let arguments = (ins
-    Optional<I1>:$if_expr
+    Optional<I1>:$if_var
```

Comment:
`if_expr` sounds more appropriate since it is not necessarily a variable (like for `private`), but can be a (constant/non-constant) expression.